locales/da.yml

```diff
@@ -43,7 +43,7 @@ da:
     blog_headline: "Seneste fra bloggen"
     contact_us: Kontaktoplysninger
     meta_description: Vi forvandler ikke-tekniske iværksætteres idéer til stabile og fremtidssikrede software løsninger, der giver overskud.
-    page_title: "Substance Lab | Programmering og vedligehold af Ruby on Rails løsninger"
+    page_title: "Programmering og vedligehold af Ruby on Rails løsninger"
```

Comment:
Argumentet for at beholde Substance Lab er , hvis man på en desktop computer har mange faner åbne, og hurtigt vil overskue hvad de indeholder. :shrug: